### PR TITLE
New package: oidentd-2.3.1

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

File: srcpkgs/oidentd/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+revision=1
+build_style=gnu-configure
+conf_files="/etc/oidentd.conf /etc/oidentd_masq.conf"
+hostmakedepends="flex"
+makedepends="libnetfilter_conntrack-devel libcap-ng-devel"
+short_desc="RFC 1413 compliant ident daemon"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-2.0-only, GFDL-1.1"
+homepage="https://oidentd.janikrabe.com/"
+#changelog="https://raw.githubusercontent.com/janikrabe/${pkgname}/v${version}/ChangeLog"
```
Comment (Johnnynator) [inline srcpkgs/oidentd/template]: You can uncomment it. xlint does accept changelog now.

File: srcpkgs/oidentd/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+pkgname=oidentd
+version=2.3.1
+revision=1
+build_style=gnu-configure
+conf_files="/etc/oidentd.conf /etc/oidentd_masq.conf"
+hostmakedepends="flex"
+makedepends="libnetfilter_conntrack-devel libcap-ng-devel"
+short_desc="RFC 1413 compliant ident daemon"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-2.0-only, GFDL-1.1"
```
Comment (cr6git) [inline srcpkgs/oidentd/template]: `GPL-2.0-or-later, GFDL-1.1-or-later`

Comment (abenson) [reply srcpkgs/oidentd/template]: Good catch, fixed

Comment (janikrabe) [reply srcpkgs/oidentd/template]: `GPL-2.0-only` was correct. Please see #1581.

**Pull request merged**